src/Mobs/Monster.cpp

```diff
@@ -397,32 +347,29 @@ void cMonster::SetPitchAndYawFromDestination()
 	{
 		if (m_Target->IsPlayer())
 		{
-			FinalDestination.y = ((cPlayer *)m_Target)->GetStance();
+			FinalDestination.y = static_cast<cPlayer *>(m_Target)->GetStance() - 1;
```

Comment:
I'm not sure what this does, but the FinalDestination.y should be the the block where the "player's knees" are at, is this the case?